data/json/itemgroups/Locations_MapExtras/mansion.json

```
@@ -1302,7 +1302,8 @@
       [ "rag", 30 ],
       [ "lycra_patch", 15 ],
       [ "nylon", 10 ],
-      { "item": "tailors_kit", "prob": 20, "charges": [ 0, 100 ] }
+      { "item": "tailors_kit", "prob": 20, "charges": [ 0, 100 ] },
+      [ "kevlar_shears", 30 ]
```

```suggestion
      [ "kevlar_shears", 5 ]
```
I think the chance of finding kevlar shears in a mansion is significantly lower than finding a tailor's kit.